rest_framework-stubs/exceptions.pyi

```diff
@@ -43,10 +43,7 @@ class APIException(Exception):
 class ValidationError(APIException):
     # ValidationError wraps `detail` in a list if it's not already a list/dict.
     detail: list[_Detail] | dict[str, _Detail]
-    default_params: ClassVar[Mapping[str, Any]]
-    def __init__(
-        self, detail: _Detail | None = ..., code: str | None = ..., params: Mapping[str, Any] | None = ...
-    ) -> None: ...
+    def __init__(self, detail: _Detail | None = ..., code: str | None = ...) -> None: ...
```

Comment:
Ugh, turns out my PR #338 was wrong. `detail` here should be `_APIExceptionInput` not `_Detail`.

Let's remove this and inherit the correct one from APIException.

```suggestion
```